### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by clarkchentw**
- Duplicate code: Same code also exist in line [1033](https://github.com/clarkchentw/hestiacp/blob/c685620abbcfb77fe0f67ac0112cda059f4a3535/install/hst-install-ubuntu.sh#L1033)
- Update Cloudflare IP range

### CODE REVIEW

**Review by jaapmarcus — COMMENTED** [review]
_It doesn't add it now any more when it doesn't exits._

Nevermind it is double...

**Comment by jaapmarcus** [comment]
We should also update existing installs for new Cloudflare ips

**Comment by clarkchentw** [comment]
> _It doesn't add it now any more when it doesn't exits._
> 
> Nevermind it is double...



> We should also update existing installs for new Cloudflare ips

**Comment by clarkchentw** [comment]
Accidently closed the PR...

- Sort the Cloudflare IP list for better matching
- Added upgrade script

File: install/upgrade/versions/1.6.0.sh [hunk 1]
```diff
@@ -67,3 +67,14 @@ fi
 if [[ ! -d $HESTIA/data/api ]]; then
     cp -rf $HESTIA_INSTALL_DIR/api $HESTIA/data/
 fi
+
+# Update Cloudflare address
+if [ -f /etc/nginx/nginx.conf ] && [ "$(grep 'set_real_ip_from 2405:8100::/32' /etc/nginx/nginx.conf)" = "" ];then
```
Comment (jaapmarcus) [inline install/upgrade/versions/1.6.0.sh:72]: You also need to update ip4 ip addresses

Comment (clarkchentw) [reply install/upgrade/versions/1.6.0.sh:72]: After sorting, there are actually no change in IPv4

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
According my list only:

`# set_real_ip_from 2405:8100::/32; was added `
+ small changes spacing

**Pull request merged**